.vsts-pipelines/builds/PRBuild.yml

```diff
@@ -0,0 +1,232 @@
+phases:
+- template: ../templates/phases/default.yml
```

Comment:
I feel like there may eventually be some contention here between the folder layout described by Arcade (`eng/common/`) and that proscribed by VSTS (`.vsts-pipelines/`).  At first glance, I think the templates you have here are ASPNet-Universe specific and not really a concern, but it's something to keep in mind.